WPMedia/fusion-news-theme-blocks
# Tmedia 398 login block without recaptcha

## Description
Add login page with basic login and password fields 

## Jira Ticket
- [TMEDIA-398](https://arcpublishing.atlassian.net/browse/TMEDIA-398)

## Acceptance Criteria
As a reader, I can log into an existing account.

Context

In order to log into an existing account, a user will need to:

Enter email address (as username)

Enter password

Click Sign In button

## Test Steps

1. Checkout this branch `git checkout TMEDIA-398-login-block-without-recaptcha`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
3. Create login page
4. Use single-column layout 
5. Add identity block 
6. Type the email and password for a preexisting account for that site 
![Screen Shot 2021-09-13 at 16 05 04](https://user-images.githubusercontent.com/5950956/133156274-5709ae7b-b238-40cc-b030-5b99d61c0582.png)


7. See the redirect 
8. Go to a homepage to ensure the authentication has happened 
![Screen Shot 2021-09-13 at 16 02 54](https://user-images.githubusercontent.com/5950956/133155988-c331917c-f88e-4ae6-9b10-1ff8b82d3913.png)


## Effect Of Changes
### Before

- No login screen 

### After

Login screen using predefined styles 

![Screen Shot 2021-09-13 at 16 05 04](https://user-images.githubusercontent.com/5950956/133156286-2d776529-4a79-4c9d-9dcf-82105036c79a.png)

## Dependencies or Side Effects

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
